### ISSUE TITLE
Cannot retrieve constraint multipliers from solution_callback

### ISSUE THREAD

**Issue by abhyshr** [LLNL/hiop#30]
I am trying to access the constraint multipliers through solution_callback, but the multipliers array `lam` (and even the constraints arrays `g`) is NULL. Below is the debugging frame which shows that g and lam passed to solution_callback are NULL.

```
Process 56591 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = breakpoint 1.1
    frame #0: 0x00000001001411f5 libopflow.dylib`OPFLOWHIOPInterface::solution_callback(this=0x000000010460a430, status=Solve_Success, n=24, xsol=0x000000010460d140, z_L=0x000000010460dd10, z_U=0x000000010460de10, m=36, g=0x0000000000000000, lam=0x0000000000000000, obj_value=5297.4067102865956) at opflow-hiop.cpp:441:18
```

This is with running the NewtonMDS solver.

I am missing setting something or does this need to be added to HIOP?

**Comment by cnpetra** [comment]
needs to be added to HiOp

**Comment by cnpetra** [comment]
as for a timeline for adding those, we're looking at 2-3 weeks

**Comment by abhyshr** [comment]
There's no hurry. I noticed this so I created an issue. Take your time, let me know once it is done.

**Comment by cnpetra** [comment]
addressed by PR #45
